linkerd/proxy/spire-client/src/client.rs

```diff
@@ -0,0 +1,63 @@
+use crate::api::{Api, SvidUpdate};
+use linkerd_error::Error;
+use linkerd_exp_backoff::ExponentialBackoff;
+use std::sync::Arc;
+use tokio::net::UnixStream;
+use tokio::sync::watch;
+use tonic::transport::{Endpoint, Uri};
+
+const UNIX_PREFIX: &str = "unix:";
+const TONIC_DEFAULT_URI: &str = "http://[::]:50051";
+
+#[derive(Clone, Debug)]
+pub struct Client {
+    socket: Arc<String>,
```

Comment:
This is a very nitpicky comment but this feels more like a socket addr rather than a socket (which I'd tend to think more of as a fd). 🤷🏻